### ISSUE TITLE
Support share session with other users?

### ISSUE THREAD

**Issue by HRNasr** [casbin/casvisor#21]

**Comment by hsluoyz** [comment]
@HRNasr is the user a logged-in Casdoor user in the same org, or just an anonymous shared link and everyone can access without login?

**Comment by HRNasr** [comment]
Yes, he is logged in as a user to the same organization with which it is possible to share the session. Of course, it is possible to see the meeting without login for the users of the organization

**Comment by hsluoyz** [comment]
@HRNasr what do you mean by `it is possible to see the meeting without login for the users of the organization` ?
